### DISCUSSION TITLE
Unable to see Raspberry pi PIco after injecting fanpico firmware

### DISCUSSION THREAD

**Discussion by BaptisteMarniere** [tjko/fanpico#88] [Q&A]
Hi,
First of all thanks for your amazing work! I'm very exciting to see the fanpico working, it could be a game changer to my daily work but I will keep you inform if everything works. 
For now I've an issue with the very beginning, I can't see my rapsberry pico after I've installed the fanpico firmware.
If i reset everything I can see it in Thommy, and i can see it in my /dev/ folders.

Maybe I'm doing something wrong.
I'm on MacOS Sonoma. I've just drag and drop the firmware fanpico-0804D-pico_w.uf2 inside the RPI-RP2 folder. After the transfert the device is ejected and after nothing...

Thanks for your help!
Best,
Baptiste

**Comment by tjko** [comment]
You have "Pico W" soldered on the fanpico board, correct?  (in case you use plain old "Pico" then you should use fanpico-0804D-pico.uf2 firmware)

Does the LED on the Pico W now blink slowly? That would indicate firmware is running ok.


I just tested on MacOS Sonoma (14.5) and fanpico showed up ok (nothing really happens on the OS other than the "usbmodem" devices show up under /dev/ directory).
In my case MacOS didn't show up prompt whether to allow or deny the USB device (I may have tried this particular pico on this machine before though...). If you get that prompt you must allow the USB device to connect....

<img width="697" alt="fanpico-macos-sonoma" src="https://github.com/tjko/fanpico/assets/1075726/de16c972-4448-401f-8a0d-c5b827729501">

**Reply by BaptisteMarniere** [reply]
Thanks for your quick answer!
I don't know what I have done with my first try but now everything works!
